### PR TITLE
CT/GU Project Global Positioning Mapped

### LINKED ISSUE #495: Concept Template with Map Coordinates
Proposal: similar to Project Global Positioning, with the following difference:

`IfcGeometricRepresentationContext` - `IfcRigidOperation` - `IfcProjectedCRS` 

Testable attributes: IMHO, at least the `IfcProjectedCRS::Name`.

### LINKED ISSUE #507: Create CT and GU for "Project Global Positioning Mapped"
I am not entirely sure what the difference is between this and the existing "Project Global Positioning"?

### PR DESCRIPTION
**Pull request by larswik**
First draft for #507 and #495

### CODE REVIEW

File: IFC4x3/ModelViews/General Usage/DocModelView.xml [hunk 1]
```diff
@@ -10514,6 +10514,13 @@ Figure 1 illustrates project relationships with spatial structures, elements, an
 						<DocTemplateItem UniqueId="e1170aa4-751f-4de7-b936-8a1a853c30d7" />
 					</Items>
 				</DocTemplateUsage>
+				<DocTemplateUsage Name="Project Global Positioning Mapped" UniqueId="225782d2-563c-445b-a7c5-396743a56a6b">
+					<Documentation>The representation context of the project refers to a global positioning, i.e. the local engineering coordinate system of the project has a mapping, including a rigid transformation, to a defined projected coordinate system (a rectangular map coordinate system, as used in GIS systems)</Documentation>
```
Comment (pjanck) [inline IFC4x3/ModelViews/General Usage/DocModelView.xml:10518]: Suggestion: I tried a bit different wording. 

```suggestion
					<Documentation>The representation context of the project refers to a global positioning, i.e. the coordinate system of the project resides within the defined projected coordinate reference system (a rectangular map coordinate system, as used in e.g. GIS systems) including a rigid transformation from/to smaller coordinate values. The representation context inherits all properties of the coordinate axes of the projected coordinate reference system.</Documentation>
```

**Review by pjanck — CHANGES_REQUESTED** [review]
I believe the documentation should be improved.

File: IFC4x3/ModelViews/General Usage/DocModelView.xml (hunk 1 above)
Comment (anborr) [inline IFC4x3/ModelViews/General Usage/DocModelView.xml:10518]: <Documentation>The representation context of the project refers to global positioning, i.e. the coordinate system of the project resides within the defined projected coordinate reference system (a map coordinate system) and is subject to a rigid transformation, e.g. to achieve smaller coordinate values. The representation context inherits all properties of the projected coordinate reference system, in particular the distortions resulting in deviations between model dimensions/distances and real-world dimensions/distances.</Documentation>

Comment (anborr) [reply IFC4x3/ModelViews/General Usage/DocModelView.xml:10518]: Polished the wording again...

Comment (anborr) [inline IFC4x3/ModelViews/General Usage/DocModelView.xml:10518]:
```suggestion
				<Documentation>The representation context of the project refers to global positioning, i.e. the coordinate system of the project resides within the defined projected coordinate reference system (a map coordinate system) and is subject to a rigid transformation, e.g. to achieve smaller coordinate values. The representation context inherits all properties of the projected coordinate reference system, in particular the distortions resulting in deviations between model dimensions/distances and real-world dimensions/distances.</Documentation>
```

**Review by anborr — CHANGES_REQUESTED** [review]
I believe the documentation should be improved.

File: IFC4x3/ModelViews/General Usage/DocModelView.xml (hunk 1 above)
Comment (larswik) [reply IFC4x3/ModelViews/General Usage/DocModelView.xml:10518]: Ok to go with the modified proposal by @anborr?

Comment (SergejMuhic) [reply IFC4x3/ModelViews/General Usage/DocModelView.xml:10518]: I would avoid using indefinite qualifiers such as "smaller". Proposal: "to truncate coordinate values".

File: IFC4x3/Templates/Project Context/Project Global Positioning Mapped/DocTemplateDefinition.xml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+									<Rules>
+										<DocModelRuleAttribute Name="TargetCRS">
+											<Rules>
+												<DocModelRuleEntity Name="IfcProjectedCRS">
+													<Rules>
+														<DocModelRuleAttribute Name="HasWellKnowTextRepresentation" Identification="WKTRepresentation">
+															<Rules>
+																<DocModelRuleEntity Name="IfcWellKnownText">
+																	<Rules>
+																		<DocModelRuleAttribute Name="WellKnownText" Identification="WKT">
```
Comment (SergejMuhic) [inline IFC4x3/Templates/Project Context/Project Global Positioning Mapped/DocTemplateDefinition.xml:20]: One thing I would suggest here is to create a partial for WKT. In order to be able to re-use it.

**Review by SergejMuhic — COMMENTED** [review]
Changes can be done in a new PR also.

**Comment by larswik** [comment]
For some reason, I couldn't commit the suggestion of @anborr directly. However, I did the change locally and pushed/committed ([15e4f12](https://github.com/bSI-InfraRoom/IFC-Specification/pull/508/commits/15e4f12c7261fe07cf462e96b2841bf354478bde)) which seemed to work. I also included the proposal from @SergejMuhic and changed the "e.g. to achieve smaller coordinate values" to "e.g. to truncate coordinate values." As I understood, we will deal with making WKT partial later. Please review again!

**Review by pjanck — APPROVED** [review]
great job!

**Review by anborr — APPROVED** [review]

**Pull request merged**